src/components/comment/CommentView.jsx

```diff
@@ -14,7 +14,7 @@ const CommentView = (props) => {
                 <span className="col-auto text-muted comment-timestamp">{timeAgo}</span>
             </div>
             <div className="row">
-                <span className="col comment-value">{props.commentValue}</span>
+                <span className="col comment-value" style={ { whiteSpace: "pre-line" } }>{props.commentValue}</span>
```

Comment:
Doesn't it make more sense to put it into CSS file ? Why do we have it here ?

Reply:
@blcham This part is outdated, it is not present in the code anymore.